flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/reader/SingleThreadMultiplexSourceReaderBase.java

```diff
@@ -81,6 +82,27 @@ public SingleThreadMultiplexSourceReaderBase(
                 context);
     }
 
+    /**
+     * The primary constructor for the source reader.
```

Comment:
`primary constructor` should only have one, but current we have two. And I think you need to update the method java notes.